pytest/tests/loadtest/locust/common/sharding.py

```diff
@@ -0,0 +1,350 @@
+import os
+import sys
+import random
+import re
+import unittest
+
+
+def char_range(lower, upper, upper_inclusive=True):
+    l = ord(lower)
+    u = ord(upper) + 1 if upper_inclusive else ord(upper)
+    return [chr(i) for i in range(l, u)]
+
+
+def alpha_num_str(length):
+    return ''.join(
+        random.choices(char_range('0', '9') + char_range('a', 'z'), k=length))
+
+
+def random_char_below(upper, upper_inclusive):
```

Comment:
ah, well I think issue with that would be that you could return two `"_"` chars in a row. Say for example the lower string is not set, and the upper string is "a_aaaa", then say we happen to generate an "a" in the first spot and a "_" in the second spot. Then we don't want the third spot to be a "_", because in that case the account ID will be invalid, so avoiding the extra "_" needs to either be handled by this function, or maybe by its caller I guess, but has to be done somewhere.